### PR TITLE
Fix checkin timing

### PR DESCRIPTION
**Pull request by justinazhang**
- Changed check in page so that instead of the check in being disabled after 1.5 seconds it's disabled until the ID is successfully or unsuccessfully scanned

### CODE REVIEW

**Comment by justinazhang** [comment]
If you have a more elegant solution besides just enabling the fields after every single case lmk haha definitely not the most pretty solution

**Comment by janiseliang** [comment]
could just add it to the showError/showSuccess functions maybe?

also i think technically the more react-y way to do the disabling would be to use a boolean state variable for disabled/not disabled. there probably isn't much difference but just for style's sake haha

File: pages/checkin.js [hunk 1]
```diff
@@ -118,17 +118,23 @@ class Checkin extends React.Component {
...
       });
     })
     .catch((err) => {
+      fieldsetCalId.disabled = false
+      e.target.calID.value = null;
+      calIdTextBox.focus();
       this.showError("Failed scanning ID: " + e.target.calID.value,300)
-    })
-  
+    })
```
Comment (janiseliang) [inline pages/checkin.js:137]: wait actually i think a `.finally()` after this line would work best

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally

**Review by janiseliang — APPROVED** [review]
woohoo

**Pull request merged**